### ISSUE TITLE
nitpick: I first expected these superscript attributions to come after the corresponding name

### ISSUE THREAD

**Issue by pastly** [ghacksuserjs/TorZillaPrint#39]
![](https://user-images.githubusercontent.com/9538998/58901979-41657b00-86d0-11e9-94a0-0a4237841aa9.png)

Thus I was really confused at first about whether you were attributing CSS inner window size or viewport size to me.

**Comment by Thorin-Oakenpants** [comment]
Good point. I guess I'm too used to the reporting style I see/use (e.g in bugzilla, or in my user.js)
```
blah blah bala see [1] blah blah
[1] link-to-ref
```

But this isn't that and I'll flip it :+1:

**Comment by Thorin-Oakenpants** [comment]
PS: Thanks for the update to your [post](https://matt.traudt.xyz/posts/how-css-alone-YF4ciVY6.html)

Also, touching on your email re entropy / anonymity score - I'm like you: sites like panopticlick, amiunique etc are fine for returning results, but their tests are limited and the datasets tainted, which in turn screws with their entropy values. And their sets can be long lived (Panopticlick recently changed to last 45 days or something)

Instead, if you look at it from the viewpoint of math in an enforced set (RFP, Tor Browser), then none of that matters. The only thing that matters is to increase the number of TB users.

Example: DNT is disabled via JS (navigator) and in the HTTP header in Tor Browser. Thus all TB users report the same value. Entropy does not exist. If you visit Panopticlick, it likes to think it's about 50/50 (even though there are three values). Even this "real world" value is bunk: DNT is opt-in (it's part of the standard), and in no-one has half the world turned on DNT.

Applying a full world assessment against a closed enforced set such as TB just makes the whole thing a bunch of lies. It's the nature of the site, repeat visitors, constantly tweaking and re-testing, attracting those who want to lower entropy => self-perpetuating myth over some values

Anyway, truck on, and thanks for your work.

PS: When I contacted you, I mixed you up with user [system33-](https://old.reddit.com/user/system33-) .. who often posts some boilerplate answers like [this one](https://old.reddit.com/r/TOR/comments/bwqt9b/tor_browser_ads_and_plugins_like_ublock/epzkbom/) with links to your blog
